src/Windows.Core/SIZE_T.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// The maximum number of bytes to which a pointer can point.
+    /// Use for a count that must span the full range of a pointer.
+    /// </summary>
+    /// <remarks>
+    /// This struct is intended for P/Invoke marshaling and must exactly match
+    /// the size of <see cref="IntPtr"/>
+    /// </remarks>
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
nit: please drop this attribute. It's the C# default. Per (a relatively recent change in) our [contribution guidelines](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#structs).